kernel/src/sched/sched_class/mod.rs

```diff
@@ -303,8 +303,12 @@ impl ClassScheduler {
                     .filter(|&cpu| cpu.as_usize() as isize <= last_chosen),
             );
         for candidate in affinity_iter {
-            let rq = self.rqs[candidate.as_usize()].lock();
-            let (load, _) = rq.nr_queued_and_running();
+            let PerCpuLoadStats {
+                queued_non_idle,
+                running_non_idle,
+            } = self.rqs[candidate.as_usize()].lock().load_stats();
+            let running = u32::from(running_non_idle.is_some());
```

Comment:
Add a comment here to justify the heuristics here:

```rust
// A penalty to a CPU running the idle task.
//
// We increase the load by one when a candidate CPU is running the idle task.
// This is shown to be a good heuristics by experiments (https://github.com/asterinas/asterinas/pull/2426).
// Of course. This heuristics is trivial and should be replaced with a more advanced one in the future.
let idle_running_penalty = u32::from(running_non_idle.is_some());
```

Reply:
Sorry. I misread the code. The above code does the opposite of what you are doing. Ignore it.